app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,18 @@
+class FoodsController < ApplicationController
+  def index
+    @facade = FoodFacade.new(params[:q])
+    @food = params[:q]
```

Comment:
Since you pass `params[:q]` into your facade and set it equal to an instance variable there, you could just add an `attr_reader` for `food` to your facade instead of having 2 instance variables here in your controller. Then, in the view, you'd do `@facade.food` on line 1. This is a pretty minor thing, but it would be easy to change and cleans up the controller a tiny bit.